keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/DoubleGreaterThanOrEqualToMaskVertex.java

```diff
@@ -0,0 +1,21 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.binary;
+
+import io.improbable.keanu.annotation.ExportVertexToPythonBindings;
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.LoadVertexParam;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+
+public class DoubleGreaterThanOrEqualToMaskVertex extends DoubleBinaryOpVertex {
+
+    @ExportVertexToPythonBindings
+    public DoubleGreaterThanOrEqualToMaskVertex(@LoadVertexParam(LEFT_NAME) DoubleVertex left,
+                                                @LoadVertexParam(RIGHT_NAME) DoubleVertex right) {
+        super(left, right);
```

Comment:
do you need a `checkHasOneNonLengthOneShapeOrAllLengthOne` here?

also in `DoubleLessThanOrEqualToMaskVertex`

Reply:
No, the super constructor (in `DoubleBinaryOpVertex` handles that)